src/default-theme/variables.css

```diff
@@ -93,6 +93,8 @@ all of MD as it is not optimized for dense, information rich UIs.
   --jp-code-padding: 5px;
   --jp-code-font-family: monospace;
 
+  --jp-notebook-scrollPadding: 100px;
```

Comment:
We have been separating the CSS attribute (`padding`) from the other parts using  a `-`, so:

`--jp-notebook-scroll-padding`